yarn.lock

```diff
@@ -1554,6 +1554,17 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-card@^1.0.0-beta.104":
```

Comment:
This shouldn't be here. What's the output of `yarn why @fluentui/react-card` ?

Reply:
```code
- Hoisted from "_project_#@fluentui#react-card"
- Hoisted from "_project_#vr-tests#@fluentui#react-card"
- Hoisted from "_project_#@fluentui#react-examples#@fluentui#react-card"
```